test/utils/test_file_utils.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+from pathlib import Path
+import shutil
+from fruit_classifier.utils.file_utils import copytree
+import os
+
+
+class TestFileUtils(unittest.TestCase):
+
+    def setUp(self):
+        self.root_dir = Path(__file__).absolute().parents[1]
+        self.src_path = self.root_dir.joinpath('test_data')
+        self.dst_path = self.root_dir.joinpath('test_data_copy')
+
+        if not self.dst_path.is_dir():
+            self.dst_path.mkdir(parents=True, exist_ok=True)
+
+    def test_copytree(self):
+        self.compare_folders()
+        self.compare_folders()
+
+    def compare_folders(self):
+        copytree(self.src_path, self.dst_path)
+
+        self.assertTrue(self.dst_path.is_dir(), 'The copied files does not exist')
```

Comment:
It's considered good practice to have the `self.assert*` in `def test_`

Reply:
Ok, and I can see why. This makes it eadier to isolate help functions outside the test lass